### PR TITLE
add output dir option to kmod 

### LINKED ISSUE #239: add output dir option to kmod
add an optional outputdir option to the kmod analysis,
if no dir is provided, the inputdir should still be used

test could then be adapted to remove suhtil.rmtree and use pytest tmp_dir instead

### PR DESCRIPTION
**Pull request by mihofer**
Fixes #239

### CODE REVIEW

**Comment by Mael-Le-Garrec** [comment]
For consistency issues you could use `outputdir` instead of `output_directory`

File: omc3/run_kmod.py [hunk 1]
```diff
@@ -33,6 +33,9 @@ def kmod_params():
                          type=str,
                          required=True,
                          help='path to working directory with stored KMOD measurement files',)
+    parser.add_parameter(name='outputdir',
+                         type=str,
```
Comment (Mael-Le-Garrec) [inline omc3/run_kmod.py:37]: why not `type=pathlib.Path` ?

**Review by Mael-Le-Garrec — DISMISSED** [review]

File: omc3/run_kmod.py (hunk 1 above)
Comment (mihofer) [reply omc3/run_kmod.py:37]: switching everything to Pathlib (and testing everything in the background behaves the same) is a separate issue #238 and i think not in the spirit of what this quick fix PR is supposed to do

**Comment by fsoubelet** [comment]
I have implemented this already in the github actions branch to fix builds. How should we proceed?

**Comment by fsoubelet** [comment]
Implemented in #230, will close this once #230 is merged

**Pull request closed without merging**